### ISSUE TITLE
RWR only working with Aircraft started hot

### ISSUE THREAD

**Issue by KingSharkoon** [Kinkkujuustovoileipa/uh-60l#57]
i only get symbology and Sound of the RWR if the Aircraft is Hot started - with a cold Start the instrument is INOP

**Comment by Kinkkujuustovoileipa** [comment]
Did some investigation into this, I believe it is partially an ED issue - there is a reference elsewhere to RWR requiring engines to be on. However why it doesn't work even with the engines on is still a mystery :(

**Comment by KingSharkoon** [comment]
Hmm, okay - maybe we can ask ED about an idea ;)

**Comment by ItsFuzeOrDart** [comment]
Yes RWR only on hot start. Also the Radios only work on hot start (yes easy coms on).

**Comment by Jedi1-4** [comment]
Rwr works correctly from cold start in 1.2.
